ext/winevt/winevt_utils.cpp

```diff
@@ -885,14 +885,20 @@ render_system_event(EVT_HANDLE hEvent, BOOL preserve_qualifiers, BOOL preserveSI
       if (preserveSID_p) {
         rbstr = rb_utf8_str_new_cstr(pwsSid);
         rb_hash_aset(hash, rb_str_new2("UserID"), rbstr);
-        LocalFree(pwsSid);
       }
-      if (ExpandSIDWString(pRenderedValues[EvtSystemUserID].SidVal,
-                       &expandSID) == 0) {
-        rbstr = rb_utf8_str_new_cstr(expandSID);
-        free(expandSID);
-        rb_hash_aset(hash, rb_str_new2("User"), rbstr);
+      /* S-1-15-3- is used for capability SIDs. So, we need to skip
+       * SID translation.
+       * See also: https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers
```

Comment:
It seems that the following article is more suitable to indicate that these SIDs should be skipped:

https://learn.microsoft.com/en-us/troubleshoot/windows-server/windows-security/sids-not-resolve-into-friendly-names

```
By design, a capability SID does not resolve to a friendly name.
```

Could you add this URL too in this comment?

Reply:
Wow, thanks for searching more suitable article. I"ll add it on this PR.

Reply:
Added the suggested link: https://github.com/fluent-plugins-nursery/winevt_c/pull/48/commits/f09c8776dc4ad60bb3d04a4b29720d6dbcf87fda
But the first one is also useful for overview of SIDs which are including ordinary ones.
So, I added simply the suggested one.

Reply:
Thanks!